lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -37,6 +37,13 @@ const ignoredPathComponents = [
   '/test/',
   '-test.js',
   '-renderer.js',
+  // Windows
+  '\\.git',
+  '\\scripts',
+  '\\node_modules',
```

Comment:
hello, fellow windows user
https://github.com/GoogleChrome/lighthouse/pull/9406

Reply:
* sigh * yeah.  This breaks anyway.  Renames everything "\\path\\to\\file" in the ctc locales.